packages/core/src/services/ai/index.ts

```diff
@@ -79,6 +79,7 @@ export async function ai({
   const m = createProvider({ provider, apiKey, config })(model)
 
   const commonOptions = {
+    ...config,
```

Comment:
careful this will include schema and output to all requests to ai

Reply:
output?

Reply:
If `schema` exists in the config object, we're replacing it afterwards anyways, so no problem

Reply:
Even though this is correct I'm worried this is a gun waiting to be fired on our foot, could you exclude schema and output props from this config before doing the spread? just to be on the safe side